### PR TITLE
[EDU-5149] feature: add Automated user provisioning with Microsoft Entra guide

### PR DESCRIPTION
**Pull request by MarianaAguilera**
### Related issue:

[EDU-5149]

### Changes

- Add Automated user provisioning with Microsoft Entra guide EN/PT.
- Add the link to the Guides page EN/PT.

### Additional links

https://github.com/aziontech/docs/pull/1173

[EDU-5149]: https://aziontech.atlassian.net/browse/EDU-5149?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by luizcunhaazion — APPROVED** [review]

File: src/content/docs/en/pages/guides/idp/microsoft-entra-user-provisioning.mdx [hunk 1]
```diff
@@ -0,0 +1,195 @@
...
+namespace: documentation_sso_microsoft_entra_provisioning
+permalink: /documentation/products/guides/microsoft-entra-automated-user-provisioning/
+---
+
+Enabling **Automated User Provisioning** streamlines identity and access management by automatically creating, updating, and deleting user accounts across applications based on changes in Microsoft Entra ID. These changes are reflected on Azion's platform users and teams, once the feature is enabled. The automation uses the SCIM protocol, making the provisioning process both scalable and secure.
+
+It simplifies lifecycle management, reduces manual work, and enhances compliance, ensuring users have appropriate access to the resources as needed.
+
+This guide will cover how to enable Microsoft Entra Automated User Provisioning for Azion's platform.
+
```
Comment (MarianaAguilera) [inline src/content/docs/en/pages/guides/idp/microsoft-entra-user-provisioning.mdx:14]:
```suggestion

:::note
This feature is available to customers with **Enterprise** and **Mission Critical** [Service Plans](https://www.azion.com/en/professional-services/).
:::
```

File: src/content/docs/pt-br/pages/guias/idp/provisionamento-usuarios-microsoft-entra.mdx [hunk 2]
```diff
@@ -0,0 +1,195 @@
...
+namespace: documentation_sso_microsoft_entra_provisioning
+permalink: /documentacao/produtos/guias/microsoft-entra-provisionamento-automatizado-de-usuarios/
+---
+
+Habilitar o **Provisionamento Automatizado de Usuários** simplifica a gestão de identidade e acesso ao criar, atualizar e excluir automaticamente contas de usuário em aplicações com base nas mudanças no Microsoft Entra ID. Essas mudanças são refletidas nos usuários e equipes da plataforma da Azion, uma vez que o recurso é habilitado. A automação utiliza o protocolo SCIM, tornando o processo de provisionamento escalável e seguro.
+
+Isso simplifica a gestão, reduz o trabalho manual e melhora a conformidade, garantindo que os usuários tenham acesso apropriado aos recursos conforme necessário.
+
+Este guia abordará como habilitar o provisionamento automatizado de usuários do Microsoft Entra para a plataforma da Azion.
+
```
Comment (MarianaAguilera) [inline src/content/docs/pt-br/pages/guias/idp/provisionamento-usuarios-microsoft-entra.mdx:14]:
```suggestion

:::note
Este recurso está disponível para clientes com [planos de serviço](https://www.azion.com/pt-br/servicos-profissionais/) **Enterprise** e **Mission Critical**.
:::
```

File: src/content/docs/en/pages/guides/idp/microsoft-entra-user-provisioning.mdx [hunk 3]
```diff
@@ -0,0 +1,198 @@
...
+:::note
+This feature is available to customers with **Enterprise** and **Mission Critical** [Service Plans](https://www.azion.com/en/professional-services/).
+:::
+---
```
Comment (MarianaAguilera) [inline src/content/docs/en/pages/guides/idp/microsoft-entra-user-provisioning.mdx:18]:
```suggestion

---
```

File: src/content/docs/pt-br/pages/guias/idp/provisionamento-usuarios-microsoft-entra.mdx [hunk 4]
```diff
@@ -0,0 +1,198 @@
...
+:::note
+Este recurso está disponível para clientes com [planos de serviço](https://www.azion.com/pt-br/servicos-profissionais/) **Enterprise** e **Mission Critical**.
+:::
+---
```
Comment (MarianaAguilera) [inline src/content/docs/pt-br/pages/guias/idp/provisionamento-usuarios-microsoft-entra.mdx:18]:
```suggestion

---
```

**Review by gabriel-azion — APPROVED** [review]

**Review by GabrielAzion — APPROVED** [review]

File: src/content/docs/en/pages/guides/idp/microsoft-entra-user-provisioning.mdx [hunk 5]
```diff
@@ -0,0 +1,199 @@
...
+You need an Azion personal token to execute the following tasks. Read the documentation to [create a new personal token](/en/documentation/products/guides/personal-tokens/).
+
+You also need to previously [configure Microsoft Entra as an Identity Provider (IdP) for Azion](/en/documentation/products/guides/sso-microsoft-entra-saml/).
+:::
+
+1. Go to your preferred API development and testing tool. For example, Postman or Swagger.
+2. Run the following `GET` request, replacing [Token value] with your Azion personal token, to obtain the current status of your IdP and its identifier on Azion.
+
+```bash
+curl --location 'https://api/azion.com/v4/iam/identity_providers' \
```
Comment (MarianaAguilera) [inline src/content/docs/en/pages/guides/idp/microsoft-entra-user-provisioning.mdx:33]:
```suggestion
curl --location 'https://api.azionapi.net/iam/identity_providers' \
```

File: src/content/docs/en/pages/guides/idp/microsoft-entra-user-provisioning.mdx [hunk 6]
```diff
@@ -0,0 +1,199 @@
...
+    - Copy the `uuid` to use it in the next steps.
+
+:::note 
+The IdP must be active and linked with the right SAML Application in the Microsoft Entra console. Check the guide on using [Microsoft Entra custom SAML app as an IdP for Azion](/en/documentation/products/guides/sso-microsoft-entra-saml/) for more details on this configuration.
+:::
+
+4. Run the following `GET` request, replacing [Token value] with your Azion personal token and the `uuid` with the identifier you got in the previous steps, to obtain more information about the IdP integration on Azion.
+
+```bash
+curl --location 'https://api/azion.com/v4/iam/identity_providers/saml2/<uuid>' \
```
Comment (MarianaAguilera) [inline src/content/docs/en/pages/guides/idp/microsoft-entra-user-provisioning.mdx:62]:
```suggestion
curl --location 'https://api.azionapi.net/iam/identity_providers/saml2/<uuid>' \
```

File: src/content/docs/pt-br/pages/guias/idp/provisionamento-usuarios-microsoft-entra.mdx [hunk 7]
```diff
@@ -0,0 +1,199 @@
...
+Você precisa de um token pessoal da Azion para executar as seguintes tarefas. Leia a documentação para [criar um novo token pessoal](/pt-br/documentacao/produtos/guias/personal-tokens/).
+
+Você também precisa [configurar o Microsoft Entra como um Provedor de Identidade (IdP) para a Azion](/pt-br/documentacao/produtos/guias/sso-microsoft-entra-saml/) previamente.
+:::
+
+1. Vá para a sua ferramenta preferida de desenvolvimento e teste de API. Por exemplo, Postman ou Swagger.
+2. Execute a seguinte requisição `GET`, substituindo [Token value] pelo seu token pessoal da Azion, para obter o status atual do seu IdP e seu identificador na Azion.
+
+```bash
+curl --location 'https://api/azion.com/v4/iam/identity_providers' \
```
Comment (MarianaAguilera) [inline src/content/docs/pt-br/pages/guias/idp/provisionamento-usuarios-microsoft-entra.mdx:33]:
```suggestion
curl --location 'https://api.azionapi.net/iam/identity_providers' \
```

File: src/content/docs/pt-br/pages/guias/idp/provisionamento-usuarios-microsoft-entra.mdx [hunk 8]
```diff
@@ -0,0 +1,199 @@
...
+  - Copie o `uuid` para usá-lo nos próximos passos.
+
+:::note 
+O IdP deve estar ativo e vinculado à aplicação SAML correta no console do Microsoft Entra. Confira o guia sobre como usar [Microsoft Entra custom SAML app as an IdP for Azion](/pt-br/documentacao/produtos/guias/sso-microsoft-entra-saml/) para mais detalhes sobre essa configuração.
+:::
+
+4. Execute a seguinte requisição `GET`, substituindo [Token value] pelo seu token pessoal da Azion e o `uuid` pelo identificador que você obteve nos passos anteriores, para obter mais informações sobre a integração do IdP na Azion.
+
+```bash
+curl --location 'https://api/azion.com/v4/iam/identity_providers/saml2/<uuid>' \
```
Comment (MarianaAguilera) [inline src/content/docs/pt-br/pages/guias/idp/provisionamento-usuarios-microsoft-entra.mdx:62]:
```suggestion
curl --location 'https://api.azionapi.net/iam/identity_providers/saml2/<uuid>' \
```

File: src/content/docs/pt-br/pages/guias/idp/provisionamento-usuarios-microsoft-entra.mdx [hunk 9]
```diff
@@ -0,0 +1,199 @@
...
+"scim_url": null
+}
+```
+
+    - `scim_integration` refere-se ao status atual da ativação do protocolo SCIM. Neste caso, `false` indica que o protocolo ainda não está ativo para funcionar com o IdP.
+
+6. Execute a seguinte requisição `PATCH`, substituindo [Token value] pelo seu token pessoal da Azion e o `uuid` pelo identificador que você obteve nos passos anteriores, para atualizar o status da sua integração SCIM do IdP.
+
+```bash
+curl --location 'https://api/azion.com/v4/iam/identity_providers/saml2/<uuid>' \
```
Comment (MarianaAguilera) [inline src/content/docs/pt-br/pages/guias/idp/provisionamento-usuarios-microsoft-entra.mdx:89]:
```suggestion
curl --location 'https://api.azionapi.net/iam/identity_providers/saml2/<uuid>' \
```

File: src/content/docs/en/pages/guides/idp/microsoft-entra-user-provisioning.mdx [hunk 10]
```diff
@@ -0,0 +1,199 @@
...
+"scim_url": null
+}
+```
+
+    - `scim_integration` refers to the current status of the SCIM protocol activation. In this case, `false` indicates that the protocol isn't active yet to work with the IdP.
+
+6. Run the following `PATCH` request, replacing [Token value] with your Azion personal token and the `uuid` with the identifier you got in the previous steps, to update the status of your IdP SCIM integration.
+
+```bash
+curl --location 'https://api/azion.com/v4/iam/identity_providers/saml2/<uuid>' \
```
Comment (MarianaAguilera) [inline src/content/docs/en/pages/guides/idp/microsoft-entra-user-provisioning.mdx:89]:
```suggestion
curl --location 'https://api.azionapi.net/iam/identity_providers/saml2/<uuid>' \
```

File: src/content/docs/en/pages/guides/idp/microsoft-entra-user-provisioning.mdx [hunk 11]
```diff
@@ -0,0 +1,199 @@
...
+You need an Azion personal token to execute the following tasks. Read the documentation to [create a new personal token](/en/documentation/products/guides/personal-tokens/).
+
+You also need to previously [configure Microsoft Entra as an Identity Provider (IdP) for Azion](/en/documentation/products/guides/sso-microsoft-entra-saml/).
+:::
+
+1. Go to your preferred API development and testing tool. For example, Postman or Swagger.
+2. Run the following `GET` request, replacing [Token value] with your Azion personal token, to obtain the current status of your IdP and its identifier on Azion.
+
+```bash
+curl --location 'https://api.azionapi.net/iam/identity_providers' \
```
Comment (MarianaAguilera) [inline src/content/docs/en/pages/guides/idp/microsoft-entra-user-provisioning.mdx:33]: pode validar a URL do endpoint, por gentileza? @luizcunhaazion

File: src/content/docs/en/pages/guides/idp/microsoft-entra-user-provisioning.mdx [hunk 12]
```diff
@@ -0,0 +1,199 @@
...
+    - Copy the `uuid` to use it in the next steps.
+
+:::note 
+The IdP must be active and linked with the right SAML Application in the Microsoft Entra console. Check the guide on using [Microsoft Entra custom SAML app as an IdP for Azion](/en/documentation/products/guides/sso-microsoft-entra-saml/) for more details on this configuration.
+:::
+
+4. Run the following `GET` request, replacing [Token value] with your Azion personal token and the `uuid` with the identifier you got in the previous steps, to obtain more information about the IdP integration on Azion.
+
+```bash
+curl --location 'https://api.azionapi.net/iam/identity_providers/saml2/<uuid>' \
```
Comment (MarianaAguilera) [inline src/content/docs/en/pages/guides/idp/microsoft-entra-user-provisioning.mdx:62]: pode validar a URL do endpoint, por gentileza? @luizcunhaazion

File: src/content/docs/en/pages/guides/idp/microsoft-entra-user-provisioning.mdx [hunk 13]
```diff
@@ -0,0 +1,199 @@
...
+"scim_url": null
+}
+```
+
+    - `scim_integration` refers to the current status of the SCIM protocol activation. In this case, `false` indicates that the protocol isn't active yet to work with the IdP.
+
+6. Run the following `PATCH` request, replacing [Token value] with your Azion personal token and the `uuid` with the identifier you got in the previous steps, to update the status of your IdP SCIM integration.
+
+```bash
+curl --location 'https://api.azionapi.net/iam/identity_providers/saml2/<uuid>' \
```
Comment (MarianaAguilera) [inline src/content/docs/en/pages/guides/idp/microsoft-entra-user-provisioning.mdx:89]: pode validar a URL do endpoint, por gentileza? @luizcunhaazion

File: src/content/docs/en/pages/guides/idp/microsoft-entra-user-provisioning.mdx (hunk 11 above)
Comment (luizcunhaazion) [reply src/content/docs/en/pages/guides/idp/microsoft-entra-user-provisioning.mdx:33]: Esse mesmo

File: src/content/docs/en/pages/guides/idp/microsoft-entra-user-provisioning.mdx (hunk 12 above)
Comment (luizcunhaazion) [reply src/content/docs/en/pages/guides/idp/microsoft-entra-user-provisioning.mdx:62]: ok

File: src/content/docs/en/pages/guides/idp/microsoft-entra-user-provisioning.mdx (hunk 13 above)
Comment (luizcunhaazion) [reply src/content/docs/en/pages/guides/idp/microsoft-entra-user-provisioning.mdx:89]: ok

**Review by luizcunhaazion — APPROVED** [review]

**Pull request merged**